packages/protocol/contracts/L1/provers/GuardianProver.sol

```diff
@@ -45,10 +45,7 @@ contract GuardianProver is Guardians {
 
         if (approved) {
             deleteApproval(hash);
-            bytes memory data =
-                abi.encodeCall(ITaikoL1.proveBlock, (meta.id, abi.encode(meta, tran, proof)));
-            (bool success,) = resolve("taiko", false).call(data);
-            if (!success) revert PROVING_FAILED();
```

Comment:
The original implementation catch different proving errors and convert them into PROVING_FAILED. I guess it's fine to not do so.

Reply:
Oh right, didn't think of that nuance.